recipes/cd3-boost-unit-definitions/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class PackageConan(ConanFile):
+    name = "cd3-boost-unit-definitions"
+    description = "A collection of pre-defined types and unit instances for working with Boost.Units quantities."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/CD3/BoostUnitDefinitions"
+    topics = ("physical dimensions", "header-only")
+    settings = "os", "arch", "compiler", "build_type"  # even for header only
+    no_copy_source = True  # do not copy sources to build folder for header only projects, unless, need to apply patches
+
+    @property
+    def _min_cppstd(self):
+        return 14
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "15",
+            "msvc": "14.1",
+            "gcc": "5",
+            "clang": "5",
+            "apple-clang": "5.1",
+        }
+
+    def export_sources(self):
+        pass
+
+    def layout(self):
+        basic_layout(self)
+
+    def requirements(self):
+        self.requires("boost/1.72.0", transitive_headers=True)
```

Comment:
why such old version?

Reply:
Good question. It's just the version I have been using in my projects. It really does not require a specific version, any of the version in conancenter will work. Should I make this a version range?

Reply:
Updated this to
```
self.requires("boost/[>=1.69.0]", transitive_headers=True)
```